code/modules/interface/keybind_style.dm

```diff
@@ -188,7 +190,8 @@ var/global/list/datum/keybind_style/keybind_styles = null
 	"X" = "swaphand",
```

Comment:
`"X" = "usehand"` will conflict with this for tg layout, you need to rebind it for tg layout

Reply:
"N" = "usehand" is rebound below for TG

Reply:
oh yeah the hotkey system i literally wrote myself works that way doesn't it lol